spec/v3/services/env_vars/create_spec.rb

```diff
@@ -58,7 +58,6 @@
         '@type' => 'env_var',
         '@representation' => 'standard',
         'name' => 'FOO',
-        'value' => 'bar',
```

Comment:
this corresponds to the current V2 implementation that does not return non-public values.